sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,20 +1,98 @@
+- These credentials now have a new `authentication()` method, which always triggers user interaction.
+
+> Read more about controlling user interaction in our [samples](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md#control-user-interaction).
+
+A successful call to `authenticate()` will return an `AuthenticationRecord`, which can be provided to new instances of these credentials to reuse previously obtained access tokens, avoiding further user interaction.
+
+> Read more about persisting user authentication data in our [samples](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/samples/AzureIdentityExamples.md#persist-user-authentication-data).
+
+#### Client credentials
+
+Identity v2 adds better support for regional endpoints on the `ClientSecretCredential` and `ClientCertificateCredential` credentials. These credentials have a new `regionalAuthority` optional property on their constructors. Additionally, a new `enum` type with all of the Azure regions is available as `RegionalAuthority`.
+
+The Azure region can also be specified through the `AZURE_REGIONAL_AUTHORITY_NAME` environment variable. If instead of a region, `AutoDiscoverRegion` is specified as the value for `regionalAuthority`, MSAL will be used to attempt to discover the region.
+
+#### Managed Identity credential
+
+In Identity v2, the `ManagedIdentityCredential` retries with exponential back-off when a request for a token fails with a 404 status code. This change only applies to environments with available IMDS endpoints.
+
+Azure Service Fabric support hasn't been added on the initial version 2 of Identity. Follow the development of this feature through [this GitHub issue](https://github.com/Azure/azure-sdk-for-js/issues/12420).
+
+#### Other changes
+
+- The Node.js version of `InteractiveBrowserCredential` has [Proof Key for Code Exchange (PKCE)](https://datatracker.ietf.org/doc/html/rfc7636) enabled by default.
+- `InteractiveBrowserCredential` has a new `loginHint` constructor option, which allows a username to be pre-selected for interactive logins.
+- In `AzureCliCredential`, we allow specifying a `tenantId` in the parameters through the `AzureCliCredentialOptions`.
+- A new error, named `AuthenticationRequiredError`, has been added. This error shows up when a credential fails to authenticate silently.
+- Errors and logged exceptions may point to the new [troubleshooting guidelines](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/identity/identity/Troubleshooting.md).
+
 ### Breaking Changes
 
+#### Breaking Changes from v1
+
+- On all of the credentials we're providing, the initial authentication attempt in the lifetime of your app will include an additional request to first discover relevant endpoint metadata information from Azure.
```

Comment:
Why is this under breaking changes?

Reply:
We have had users complaining about unexpected network requests in the past. It’s a behavioral change that might break them. It did break our tests for a bit.

Reply:
I’ll move it to other changes 🤔